drivers/ng_at86rf2xx/ng_at86rf2xx_netdev.c

```diff
@@ -293,7 +293,8 @@ static void _receive_data(ng_at86rf2xx_t *dev)
     /* fill missing fields in netif header */
     netif = (ng_netif_hdr_t *)hdr->data;
     netif->if_pid = dev->mac_pid;
-    ng_at86rf2xx_rx_read(dev, &(netif->lqi), 1, pkt_len);
+    /* LQI offset: Payload (pkt_len) + FCS (2 byte) */
+    ng_at86rf2xx_rx_read(dev, &(netif->lqi), 1, pkt_len + 2);
```

Comment:
`+ 2` doesn't hold for the rf231 as 

> If the SRAM read access is used to read an RX frame, the frame length field (PHR) cannot be accessed.

Compare section 9.3.2 User accessible Frame Content of the at86rf231 reference.
![screen shot 2015-07-06 at 11 03 08](https://cloud.githubusercontent.com/assets/56618/8518903/a8b3aee4-23ce-11e5-8cbb-d9d5dcf945fa.png)